builder/lxc/config.go

```diff
@@ -65,6 +73,10 @@ func NewConfig(raws ...interface{}) (*Config, error) {
 		errs = packer.MultiErrorAppend(errs, fmt.Errorf("Failed parsing init_timeout: %s", err))
 	}
 
+	if c.LxcTemplate.Name != "" && c.RootFs != (RootFsConfig{}) {
+		errs = packer.MultiErrorAppend(errs, fmt.Errorf("Cannot build with both lxc_template and rootfs configuration options"))
```

Comment:
Fair enough, I'll gladly take your word for it as long as there was some consideration for it.